.github/workflows/autobuild.yaml

```
@@ -142,28 +142,28 @@ jobs:
       - name: Check if tag exists
         uses: mukunku/tag-exists-action@v1.6.0
         id: check-tag
-        with: 
-          tag: "${{steps.get-version.outputs.version}}"
+        with:
+          tag: $env:version

```

```suggestion
          tag: ${{ env.version }}
```
`$env:version` is Powershell syntax, not GitHub Workflow syntax.